app/views/users/_edit_form.html.erb

```diff
@@ -41,7 +41,8 @@
         <%= f.password_field :password, { placeholder: I18n.t('users._form.password'),
                                           tabindex: 4,
                                           class: 'form-control',
-                                          id: 'password' }
+                                          id: 'password',
+                                          onpaste: 'return false;' }
```

Comment:
>>not in _edit_form to avoid conflicts

But you have done changes here.

Reply:
not placeholder one

Reply:
because I'll be doing that in this https://github.com/publiclab/plots2/pull/4386